front/src/modules/ui/input/autosize-text/components/__stories__/AutosizeTextInput.stories.tsx

```diff
@@ -14,3 +14,7 @@ export default meta;
 type Story = StoryObj<typeof AutosizeTextInput>;
 
 export const Default: Story = {};
+
+export const ButtonVariant: Story = {
+  args: { variant: 'button' },
```

Comment:
let's also add Default + Catalog for this one (see Chip.stories.tsx for example)